data/mods/Magiclysm/Spells/animist.json

```
@@ -186,5 +186,32 @@
     "spell_class": "ANIMIST",
     "energy_source": "MANA",
     "flags": [ "PERMANENT", "NO_LEGS", "CONCENTRATE" ]
+  },
+  {
+    "id": "stormshaper_summon_wisps",
```

```suggestion
    "id": "summon_wisps",
```
or animist, but i don't particularly have a strong opinion about the prefixes matching the classes